### ISSUE TITLE
OverflowError: cannot convert float infinity to integer

### ISSUE THREAD

**Issue by RamanGouda** [naokishibuya/car-finding-lane-lines#2]
I am trying to execute this code and i am getting the following error
Warning (from warnings module):
  File "E:\Spring2018\MastersProject\ProjectFiles\hslColorMas_CMP.py", line 123
    x1 = int((y1 - intercept)/slope)
RuntimeWarning: divide by zero encountered in double_scalars

**Comment by marcoag** [comment]
This is a problem that comes due to the different ways of managing the division between python2 and python3. I assume you are using python2, therefore all your slopes are coming as either 0 or -1. This is due to the fact that python2 will output the division result between integer numbers as an integer (floor division). Instead, when using python3 the result of integer numbers division might be a float (true division). See the following example:

Python 2:
```
1/ 2         #integer truncation (floor division)
0
```
Python 3:
```
>>> 1/ 2      #returns real quotient (true division)
0.5
```

Long story short you can either switch to python3 or just update the slope calculation as follows, so you get a float out of it (change this line in the average_slope_intercept function):

slope = (y2-y1)/float(x2-x1)
